src/Unicorn/Predicates/PredicateResult.cs

```diff
@@ -15,5 +15,6 @@ public PredicateResult(string justification)
 
 		public bool IsIncluded { get; private set; }
 		public string Justification { get; private set; }
+		public PresetTreeRoot PresetTreeRoot { get; set; }
```

Comment:
The predicate interface does not force the usage of `PresetTreeRoot`, that is an implementation detail of `SerializationPresetPredicate`. I'd prefer to see this bubble up via something more generic like a string, like `PredicateComponentId` or something - since it's just being used for display purposes.